microshift_running_apps/microshift-greenboot.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: ASSEMBLY
+[id="microshift-greenboot"]
+= The greenboot health check
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-greenboot
+
+toc::[]
+
+You might be challenged troubleshooting edge-device problems after software or an operating system upgrade fails. Edge devices are often in locations where direct serviceability is either limited or non-existent. In these cases, you can use greenboot to troubleshoot software problems for these devices. Greenboot is the Generic Health Check Framework for the `systemd` service on RPM-OSTree-based systems.
+
+Using greenboot can reduce your risk of being locked out of an edge device during upgrades and also prevent a significant interruption of service if an upgrade fails. When a failure is detected, the system boots into the last known working configuration using the `rpm-ostree` rollback capability.
+
+include::modules/microshift-install-greenboot-rpm.adoc[leveloffset=+1]
+include::modules/microshift-greenboot-healthchecks-list.adoc[leveloffset=+1]
+
+The `40_microshift_running_check.sh` health check script is also included in the `microshift-greenboot` RPM. The script validates that the required {product-title} services are running.
+//Q is the microshift script in addition to the fedora greenboot rpms, or are we giving users just the microshift-greenboot rpm?
```

Comment:
We are shipping microshift-greenboot RPM, which is an optional component. If installed, it depends on the greenboot RPM, which would be pulled implicitly.